### PR TITLE
completed express-post-json exercise - Lyn Kim

### PR DESCRIPTION
**Pull request by lyn-kim**
What does the express.json() middleware do and when would you need it?
o	It’s used to recognize incoming requests as JSON object
o	When manipulating JSON objects via terminal

express-post-json

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  res.json(grades);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  grades[nextId] = req.body;
+  req.body.id = nextId++;
+  res.json(req.body);
+  res.send().status(201);
```
Comment (thebearingedge) [inline express-post-json/index.js:21]: Be sure to set the status code before `res.json()`. The response is sent from top to bottom and the status code goes on the start line, so we need to specify the status code before the body.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great, just one small change

File: express-post-json/index.js (hunk 1 above)
Comment (lyn-kim) [reply express-post-json/index.js:21]: Ah got it. Thanks Tim

**Review by thebearingedge — CHANGES_REQUESTED** [review]
🤔 
Your POST response body should be the newly added grade object, including its id. 

```http
HTTP/1.1 201 Created
Content-Type: application/json;

{
  "id": 1,
  "name": "Lyn",
  "course": "JavaScript",
  "score": 100
}
```

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**